esx_service/utils/auth_data.py

```diff
@@ -664,7 +663,10 @@ def new_db(self):
         err = self.__create_default_tenant()
         if err:
             return err
-        err = self.__create_default_privileges()
+        err = self.__create_all_ds_privileges_for_default_tenant()
+        if err:
+            return err
+        err = self.__create_vm_ds_privileges_for_default_tenant()
```

Comment:
I think in this case, we need to catch the error, and when we hit the error, remove the DB. @msterin  Any comments?

I think the fix for the issue you pointed out is not related to this PR, and I will file a new issue to track that.